src/components/labware/Labware.tsx

```diff
@@ -144,13 +144,15 @@ const Labware = ({
   slotBuilder
 }: React.PropsWithChildren<LabwareProps>) => {
   const labwareMachine = React.useMemo(() => {
-    return createLabwareMachine({
+    return createLabwareMachine();
+  }, []);
+  const [current, send] = useMachine(labwareMachine, {
```

Comment:
Why has this context parameter been introduced?

Reply:
There was a warning before in console 'Machine given to `useMachine` has changed between renders. This is not supported and might lead to unexpected results.'
So, if we pass in parameters while creating  machine based on a component prop, machine instance will be recreated whenever this prop/parameter changes and there by changing the useMachine which accept this machine instance as param which is not recommended. This  way of creating and using the  state machine , in which we create a static instance and then defining the context of state machine using the passed in parameters which will ensure that the machine instance will not be changed.